nyc_taxis/challenges/default.json

```diff
@@ -938,14 +938,6 @@
           "warmup-iterations": 10,
           "iterations": 50
         },
-      {# non-serverless-doc-partitioning-marker-start #}{%- if build_flavor != "serverless" -%}
-        {
-          "operation": "avg_passenger_count_esql_doc_partitioning",
-          "clients": 1,
-          "warmup-iterations": 10,
-          "iterations": 50
-        },
-      {%- endif -%}{# non-serverless-doc-partitioning-marker-end #}
```

Comment:
Why did we get rid of these at the same time?

Reply:
An alternative to `pragma` removal would be a track param which defaults to `false`, perhaps that's a better idea. @elastic/es-analytical-engine Any preferences?

Reply:
You can add a URL parameter `accept_pragma_risks=true `to allow pragmas in release builds.